### PR TITLE
Install Instructions of Docker extenstion for Meshery

### LINKED ISSUE #5473: [Docs] Add description of and install instructions for Docker Extension for Meshery
#### Current State
A Docker Extension for Meshery has been created and will be publicly available at DockerCon 2022.  A description of the extension and how to install it are needed. See https://layer5.io/docker-extension-meshery and https://hub.docker.com/r/meshery/docker-extension-meshery for details.

#### Desired State
Add Docker Extension as an option on:

- https://docs.meshery.io/installation/quick-start
- https://docs.meshery.io/installation/platforms/docker-extension (a new page)

---

#### [Contributor Guides](https://docs.meshery.io/project/contributing) and Resources
- 📚 [Instructions for contributing to documentation](https://docs.meshery.io/project/contributing/contributing-docs)
   - Meshery documentation [site](https://docs.meshery.io/) and [source](https://github.com/meshery/meshery/tree/master/docs)
- 🛠 [Meshery Build & Release Strategy](https://docs.meshery.io/project/build-and-release)
- 🎨 Wireframes and designs for Meshery UI in [Figma](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI)
- 🙋🏾🙋🏼 Questions: [Layer5 Discussion Forum](https://discuss.layer5.io) and [Layer5 Community Slack](http://slack.layer5.io)

### PR DESCRIPTION
**Pull request by karnatisrinivas**
Signed-off-by: Srinivas karnati <karnatisrinivasulareddy@gmail.com>

**Description**

This PR fixes #5473 

**Notes for Reviewers**

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [X] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit 0a8855eddf638f28dc9103b71f34d3501f3bbedc at: https://62775366e0a01b07f3df8d87--meshery-docs-preview.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit 0aa8a6c3ac81ad72a0be219752da5ea236301731 at: https://62795ad498ee8c0c7e1ca8c4--meshery-docs-preview.netlify.app

File: docs/pages/installation/docker-extension.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+---
```
[the remark this answers is not in the document]
Comment (karnatisrinivas) [reply docs/pages/installation/docker-extension.md:1]: Hello Lee, I've added Docker extension to Navigation. Kindly review.

**Comment by leecalcote** [comment]
@karnatisrinivas will you add a small troubleshoot and/or FAQ section to the extension page?

TCP port 7877 should be unused and available on the user's system. // @sudo-NithishKarthik

**Comment by leecalcote** [comment]
Another issue to open up is the need to update (@sudo-NithishKarthik) the extension architecture slide (in the Meshery Architecture deck) and to create a new section in the Meshery Architecture docs (@karnatisrinivas)

File: docs/_data/toc.yml [hunk 2]
```diff
@@ -38,6 +38,8 @@
           url: installation/platforms/aks
         - title: Docker
           url: installation/platforms/docker
+        - title: Docker extension
```
Comment (leecalcote) [inline docs/_data/toc.yml:41]:
```suggestion
        - title: Docker Extension
```

**Comment by l5io** [comment]
🚀 Preview for commit 7d837b103a9f40a800146ffaf38616aacbd0bca1 at: https://62796c20936193005615f559--meshery-docs-preview.netlify.app

**Pull request merged**